build/Publish.targets

```diff
@@ -5,12 +5,14 @@
   <Import Project="$(MSBuildThisFileDirectory)/publish/PublishDebian.targets" />
   <Import Project="$(MSBuildThisFileDirectory)/publish/Badge.targets" />
   <Import Project="$(MSBuildThisFileDirectory)/publish/Checksum.targets" />
-
+  <Import Project="$(MSBuildThisFileDirectory)/publish/RuntimeCoherence.targets" />
+  
   <!-- PUBLISH_TO_AZURE_BLOB env variable set by CI -->
   <Target Name="Publish"
           Condition=" '$(PUBLISH_TO_AZURE_BLOB)' != '' "
           DependsOnTargets="Init;
                             Package;
+                            RuntimeCoherence;
```

Comment:
nit pick: all task start with verb, maybe this should be _evaluateRuntimeCoherence_